src/signals/incident-management/containers/IncidentDetail/components/StatusForm/index.js

```diff
@@ -81,23 +81,25 @@ const Notification = styled.div`
     css`
       border-left: 3px solid;
       display: block;
-      margin: 30px 0;
-      padding-left: 20px;
+      margin: ${themeSpacing(8, 0)};
+      padding-left: ${themeSpacing(5)};
       border-color: #ec0000;
     `}
 
   line-height: 22px;
 `;
 
 const Wrapper = styled(Row)`
-  padding-top: 20px;
+  background-color: ${themeColor('tint', 'level1')};
```

Comment:
White remains white, no matter which way you turn it. Replacing white with white seems a bit pointless.

Reply:
Agree but for the consistency...